code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -684,6 +693,7 @@
 	punchstunthreshold = 7
 	punchdamagehigh = 8 // not as heavy as stone
 	prefix = "Cloth"
+	special_names = list("Fiber", "Bale")

```

Comment:
cloth golems have a separate naming system with pharaoh names

Reply:
oh darn, I overlooked that one